src/MessagingInput/MessagingInput.tsx

```diff
@@ -54,6 +55,7 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
     sendButtonText = "Send",
     labelText = "Send a message",
     disableSendButton,
+    showReturnKeyInstructionsOnDesktop,
```

Comment:
feel free to let me know if you have a better name.. this might be a bit lazy :D